server/controllers/utils/buildOverviewActions.test.ts

```diff
@@ -74,30 +84,26 @@ describe('buildOverviewActions', () => {
 
   describe('Log an activity application', () => {
     test.each`
-      roles                 | activeCaseLoadId | prisonId | activitiesEnabledPrisons | prisonerStatus  | visible
-      ${[Role.ActivityHub]} | ${'MDI'}         | ${'MDI'} | ${['MDI']}               | ${'SOMETHING'}  | ${true}
-      ${[Role.ActivityHub]} | ${'LEI'}         | ${'MDI'} | ${['MDI']}               | ${'SOMETHING'}  | ${false}
-      ${[Role.ActivityHub]} | ${'MDI'}         | ${'LEI'} | ${['MDI']}               | ${'SOMETHING'}  | ${false}
-      ${[Role.ActivityHub]} | ${'MDI'}         | ${'MDI'} | ${['LEI']}               | ${'SOMETHING'}  | ${false}
-      ${[Role.ActivityHub]} | ${'MDI'}         | ${'MDI'} | ${['MDI']}               | ${'ACTIVE OUT'} | ${false}
-      ${[]}                 | ${'MDI'}         | ${'MDI'} | ${['MDI']}               | ${'SOMETHING'}  | ${false}
-    `(
-      'user can see: $visible',
-      ({ roles, activeCaseLoadId, prisonId, activitiesEnabledPrisons, prisonerStatus, visible }) => {
-        const user = { ...userMock, userRoles: roles, activeCaseLoadId }
-        const prisoner = { ...PrisonerMockDataA, status: prisonerStatus, prisonId }
+      roles                 | activeCaseLoadId | prisonId | availableServices         | prisonerStatus  | visible
+      ${[Role.ActivityHub]} | ${'MDI'}         | ${'MDI'} | ${[{ id: 'activities' }]} | ${'SOMETHING'}  | ${true}
+      ${[Role.ActivityHub]} | ${'LEI'}         | ${'MDI'} | ${[{ id: 'activities' }]} | ${'SOMETHING'}  | ${false}
+      ${[Role.ActivityHub]} | ${'MDI'}         | ${'LEI'} | ${[{ id: 'activities' }]} | ${'SOMETHING'}  | ${false}
+      ${[Role.ActivityHub]} | ${'MDI'}         | ${'MDI'} | ${[{ id: 'something' }]}  | ${'SOMETHING'}  | ${false}
+      ${[Role.ActivityHub]} | ${'MDI'}         | ${'MDI'} | ${[{ id: 'activities' }]} | ${'ACTIVE OUT'} | ${false}
+      ${[]}                 | ${'MDI'}         | ${'MDI'} | ${[{ id: 'activities' }]} | ${'SOMETHING'}  | ${false}
+    `('user can see: $visible', ({ roles, activeCaseLoadId, prisonId, availableServices, prisonerStatus, visible }) => {
```

Comment:
Is the test only checking that `activities` is present? Was expecting to see a list of prisons where activities are enabled. The /info endpoints returns ```{
"activeAgencies": [
"LEI",
"RSI",
"PVI",
"STI",
"HEI",
"LPI",
"MDI"
]
}

Reply:
The header has already done that work and made the decision if activities should be shown to the user. This is using that decision